dashboard-ui/src/components/Survery/survery.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+class Survery extends React.Component {
```

Comment:
Purely out of curiosity since I might be working on the dashboard one day, is there a reason we're mixing up the two different ways of defining components? I've seen both ways in this code base (class with render and a function (which is my preferred way)) so I just wanted to know why you choose each one when you do.

Reply:
I started working on the dashboard based off of MVP1 code and all of the components I have made I have defined this way. So the other way to define a component you are referring to was probably written by either Jacob or Brian. I am completely fine with standardizing to whatever way you prefer (I have no preference). This is just the way I learned how to define React components when I started picking up React a couple months ago.

Reply:
They can be mixed without a problem, I just didn't know if there was a reason. Since there's not, no big deal. In my opinion, functionally defined components are easier to program, read, and control and render faster. I also think that some functionality that is available in functional components is not available in class components, or is at least much easier, but I'm not trying to turn your world upside down :)

Reply:
No problem at all, thanks for pointing that out. I'm curious to look into it, especially if it your approach renders faster.

Reply:
https://react.dev/reference/react/Component This React official link recommends using functions over classes, but you should probably do some more external research on why because they don't really explain why

Reply:
I prefer the classes just on a personal note, I feel like it is easier to keep track of the state object, but then again the function way also gets you lots of functionality and I have used both before.  My 2 cents.